draft-thomson-quic-tls.md

```diff
@@ -758,6 +789,8 @@ tune this value).  A server is less likely to generate a packet reflection
 attack if the data it sends is a small multiple of the data it receives.  A
 server SHOULD use a HelloRetryRequest if the size of the handshake messages it
 sends is likely to exceed the size of the ClientHello.
+TODO: At the QUIC layer, handshake packets are required to be padded to the
+maximium packet size to be used for the connection.
```

Comment:
That seems unwise.  I know that PMTU detection is a big TODO item, but I would have thought that the most desirable approach would be to start with a large, but still conservative value.  Large enough that you don't trigger packet reflection attack mitigation code, but small enough that you clear any small MTU problems.  Then once you are going you test for a larger MTU.

Once you are fully established, checking for a larger MTU presents less of a risk to the critical early bytes of the connection.  Losing a ClientHello is painful, but a few packets here and there aren't going to be the end of the world.